postgres/organisation_service_test.go

```diff
@@ -190,4 +190,28 @@ func TestOrganisationService(t *testing.T) {
 		assert.Nil(t, err, "Getting users in organisation should not error")
 		assert.Len(t, users, 0, "Users list should be empty")
 	})
+
+	t.Run("ok: add user to organisation and check email", func(t *testing.T) {
+		name := tests.Faker.BS()
```

Comment:
Je gebruikt name verder niet dus je kan die hieronder gewoon inlinen (zoals je bij organisation2 ook doet)

Reply:
aangepast in de hele file, het kwam in heel wat tests voor